File: api/Params/Admin/TaskParams.cs
namespace api.Params.Admin
{
    public class TaskParams: PaginationParams
    {
        public int TaskId { get; set; }
        public string TaskType { get; set; }
        public DateTime TaskDate { get; set; }
        public int candidateId { get; set; }
        public int ApplicationNo { get; set; }
        public string ResumeId { get; set; }   
        public string AssignedToUserName { get; set; }
        public string AssignedByUsername { get; set; }
        public string TaskStatus { get; set; }
        public int OrderId { get; set; }

    }
}

File: client/src/app/_models/params/Admin/taskParams.ts
export class TaskParams {
     taskId=0;
     taskType = "";
     taskDate = new Date('1900-01-01');
     candidateId: number=0;
     applicationNo: number=0;
     resumeId: string='';
     assignedToUsername = "";
     taskOwnerUsername = "";
     taskStatus ='';
     orderId = 0;
     
     sort = "taskDate";
     pageNumber = 1;
     pageSize = 15;
     search: string='';
}